pyaarlo/camera.py

```diff
@@ -635,6 +635,12 @@ def _take_idle_snapshot(self):
                           headers={"xcloudId": self.xcloud_id})
 
     def request_snapshot(self):
+        """Requests a snapshot from the camera without blocking.  
+
+        The snapshot can be handled with callbacks registered to
+        LAST_IMAGE_SRC_KEY - lastImageSource starting with snapshot/, and/or
```

Comment:
I think this line should be
   `lastImageSource starting with snapshot/, or capture/`